bin/node/src/service.rs

```diff
@@ -224,7 +224,12 @@ fn get_validator_address_cache(aleph_config: &AlephCli) -> Option<ValidatorAddre
 fn get_net_config(
     config: &Configuration,
     client: &Arc<FullClient>,
-) -> (FullNetworkConfiguration, ProtocolNaming) {
+) -> (
+    FullNetworkConfiguration,
+    ProtocolNaming,
+    Box<dyn sc_network::config::NotificationService>,
+    Box<dyn sc_network::config::NotificationService>,
```

Comment:
Hm, maybe return these in a struct with named (`pub`) fields? There are two identical types here and it would be nice to be able to tell which one corresponds to which network without reading the code of the function.

Reply:
done

Reply:
Very different than what I had in mind, but definitely works. :D